build: native windows build fails

At least on windows the oxygen-patched-xerces seems to depend on oxygen-basic-utilities

On Windows 10 21H2 in CMD shell

java -version
```bash
openjdk version "11.0.18" 2023-01-17
OpenJDK Runtime Environment Temurin-11.0.18+10 (build 11.0.18+10)
OpenJDK 64-Bit Server VM Temurin-11.0.18+10 (build 11.0.18+10, mixed mode)
```

ant -version
```bash
ANT_OPTS is set to  -Djava.security.manager=allow
Apache Ant(TM) version 1.10.13 compiled on January 4 2023
```

after
```bash
git clone https://github.com/music-encoding/music-encoding.git --recursive
```

from withing the clone directory the call of
```bash
ant -lib lib/saxon/saxon-he-11.4.jar
```

fails on the canonicalize-source target with
```bash
     [java] Exception in thread "main" java.lang.NoClassDefFoundError: ro/sync/basic/io/FilePathToURI
     [java]     at org.apache.xerces.impl.XMLEntityManager.fixURI(XMLEntityManager.java:2599)
     [java]     at org.apache.xerces.impl.XMLEntityManager.expandSystemId(XMLEntityManager.java:2079)
     [java]     at org.apache.xerces.impl.XMLEntityManager.setupCurrentEntity(XMLEntityManager.java:1108)
     [java]     at org.apache.xerces.impl.XMLVersionDetector.determineDocVersion(Unknown Source)
     [java]     at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
     [java]     at org.apache.xerces.parsers.XML11Configuration.parse(Unknown Source)
     [java]     at org.apache.xerces.parsers.XMLParser.parse(Unknown Source)
     [java]     at org.apache.xerces.parsers.AbstractSAXParser.parse(Unknown Source)
     [java]     at org.apache.xerces.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
     [java]     at org.xmlresolver.cache.ResourceCache.reset(ResourceCache.java:211)
     [java]     at org.xmlresolver.cache.ResourceCache.<init>(ResourceCache.java:145)
     [java]     at org.xmlresolver.XMLResolverConfiguration.getFeature(XMLResolverConfiguration.java:1098)
     [java]     at org.xmlresolver.CatalogResolver.<init>(CatalogResolver.java:51)
     [java]     at org.xmlresolver.Resolver.<init>(Resolver.java:68)
     [java]     at net.sf.saxon.lib.CatalogResourceResolver.<init>(CatalogResourceResolver.java:46)
     [java]     at net.sf.saxon.Configuration.init(Configuration.java:367)
     [java]     at net.sf.saxon.Configuration.<init>(Configuration.java:230)
     [java]     at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
     [java]     at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
     [java]     at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
     [java]     at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
     [java]     at java.base/java.lang.Class.newInstance(Class.java:584)
     [java]     at net.sf.saxon.Configuration.newConfiguration(Configuration.java:246)
     [java]     at net.sf.saxon.s9api.Processor.<init>(Processor.java:69)
     [java]     at net.sf.saxon.Transform.doTransform(Transform.java:321)
     [java]     at net.sf.saxon.Transform.main(Transform.java:81)
     [java] Caused by: java.lang.ClassNotFoundException: ro.sync.basic.io.FilePathToURI
     [java]     at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
     [java]     at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
     [java]     at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
     [java]     ... 26 more
     ```

--- a/build.xml
+++ b/build.xml
@@ -52,6 +52,7 @@
     <property name="saxon.jar.file" value="saxon-he-11.4.jar"/>
     <property name="xerces.version" value="25.1.0.1"/>
     <property name="xerces.jar.file" value="oxygen-patched-xerces-${xerces.version}.jar"/>
+    <property name="oxygen.basic.utilites.jar.file" value="oxygen-basic-utilities-${xerces.version}.jar"/>
     <path id="mei.classpath">
         <fileset dir="lib" erroronmissingdir="false">
             <include name="**/*.jar" />
@@ -142,6 +143,7 @@
     <target name="xerces-download" unless="${xerces-available}">
         <mkdir dir="${dir.lib.xerces}"/>
         <get src="https://www.oxygenxml.com/maven/com/oxygenxml/oxygen-patched-xerces/${xerces.version}/${xerces.jar.file}" dest="${dir.lib.xerces}"/>
+        <get src="https://www.oxygenxml.com/maven/com/oxygenxml/oxygen-basic-utilities/${xerces.version}/${oxygen.basic.utilites.jar.file}" dest="${dir.lib.xerces}"/>
     </target>
 
     <target name="canonicalize-source" description="Canonicalizes the mei-source.xml, i.e. resolves xincludes and puts result in dist/mei-source_canonicalized.xml" depends="init">

